### PR TITLE
GSW-838 feat add query metadata in every api function

### PR DESCRIPTION
**Pull request by r3v4s**
# New
- every function for api call should include query metadata
- apply camelCase for json

# Change
- every dry* functions are defined using its own file

# Fix
- test case

### CODE REVIEW

**Review by notJoon — APPROVED** [review]

**Comment by jinoosss** [comment]
The function to get a single pool has been removed..

**Comment by r3v4s** [comment]
> @r3v4s , The function to get a single pool has been removed.. It was changed to an internal function, is there a reason?

we(@r3v4s @notJoon @mconcat) decided to remove every api related functions except the ones necessary for inserting data to database.

**Comment by dongwon8247** [comment]
@r3v4s Re-implement this feature. We need this because it's better to provide real-time data via RPC when providing liquidity.

**Comment by dongwon8247** [comment]
> we(@r3v4s @notJoon @mconcat) decided to remove every api related functions except the ones necessary for inserting data to database.

@r3v4s Also next time, provide a detailed explanation why it was decided that way and the benefits of doing it. Otherwise It's hard to follow up and understand.
